integration/entrypoint.sh

```diff
@@ -571,17 +596,19 @@ only_restart_snapshotter multiple
 kill_snapshotter_and_nydusd_recover shared
 kill_snapshotter_and_nydusd_recover multiple
 
-# kill_multiple_nydusd_recover_failover multiple
-# kill_multiple_nydusd_recover_failover shared
 
 ctr_snapshot_usage multiple
 ctr_snapshot_usage shared
 
 blob_manager_pull_preheat multiple fusedev
 
 if [[ $(can_erofs_ondemand_read) == 0 ]]; then
+    kill_multiple_nydusd_recover_failover multiple
+    kill_multiple_nydusd_recover_failover shared
```

Comment:
It seems that `can_erofs_ondemand_read` is used to judge if erofs is available. Why do we have `kill_multiple_nydusd_recover_failover` and  `kill_multiple_nydusd_recover_failover` which use fuse driver in this case?

Reply:
Because github action's host's kernel can't support EROFS on-demand read. So we have an internal kernel supporting both nydusd failover and erofs.

Reply:
Got it.

Reply:
But it seems that `kill_multiple_nydusd_recover_failover` can also run without EROFS?
```shell
function kill_multiple_nydusd_recover_failover {
    ...
    reboot_containerd "${daemon_mode}" fusedev failover
    ...
}
```